config/serverless.oblt.essentials.yml

```diff
@@ -2,4 +2,17 @@
 
 ## Disable xpack plugins
 xpack.infra.enabled: false
-xpack.slo.enabled: false
\ No newline at end of file
+xpack.slo.enabled: false
+xpack.apm.enabled: false
+uiSettings.overrides.defaultRoute: /app/discover
+
```

Comment:
Can you add `xpack.observabilityAIAssistant.enabled: false` to disable the AI Assistant?

Reply:
That will close https://github.com/elastic/observability-dev/issues/4498 and https://github.com/elastic/obs-ai-assistant-team/issues/257